### DISCUSSION TITLE
NOt able to save from Colab into Gihub

### DISCUSSION THREAD

**Discussion by Moznpy** [community/community#42982] [New to GitHub]
Hello!
I am kind of new to the programming world. I am trying to save a python code on colab into my github repository, but when i click save, nothing happens and I dont know where to find my files, if they are saved on github! But i dont think they were saved. 

Any advice on what I am missing ehre?

Thank you in advance.
Mozn

**Comment by radar-blip** [comment]
Where are you clicking `save` at?

Anyway, all you need to do is open a repository (on github) and upload your file(s) into that repository.
Then click on `Commit` down at the bottom of the page.
Then all your files will be in that repository.

![files](https://user-images.githubusercontent.com/39196767/210130526-d70ca7d8-8582-4de6-8bce-01ffd086ea9a.png)

**Reply by Moznpy** [reply]
Thank you very much for your reply!
